<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRegDialog</class>
 <widget class="QDialog" name="addRegDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление регистрации</string>
  </property>
  <widget class="QGroupBox" name="groupBoxauto">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>1041</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Транспортное средство</string>
   </property>
   <widget class="QComboBox" name="comboBoxtype">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Легковое ТС</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Грузовое ТС</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пассажирское ТС</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип транспортного средства</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_brand">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>92</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Марка ТС</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_year">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>132</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Год выпуска</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_vin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>172</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VIN-код</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_hp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>212</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Мощность двигателя</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_color">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>91</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_extra">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>132</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип кузова</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_brand">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>83</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_year">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>122</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_vin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>162</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hp">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>202</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_color">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>83</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_extra">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>122</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_choosevehicle">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>22</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать ТС из базы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancelvehicle">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>22</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Отменить выбор / Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>1041</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Данные о владельце</string>
   </property>
   <widget class="QLabel" name="label_typeowner">
    <property name="geometry">
     <rect>
      <x>-8</x>
      <y>30</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Категория владельца</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxowner">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Физическое лицо</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Юридическое лицо</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_fullname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>92</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fullname">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>82</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_addr">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>132</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес прописки</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_addr">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>122</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_telephone">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>172</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Телефон</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_telephone">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>162</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_chief">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>122</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_chief">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>132</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Руководитель</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phaddr">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>82</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_phaddr">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>92</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Физический адрес</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_chooseowner">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>22</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать владельца из базы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cancelowner">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>22</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Отменить выбор / Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_submit">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>530</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Добавить запись</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>811</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Гос. номер</string>
   </property>
   <widget class="QLabel" name="label_tag">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tag">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_choosetag">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать номер из базы</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
